### PR TITLE
Fixed Repeating Linear Gradients

### LINKED ISSUE #59: Unexpected behavior for repeating-linear-gradient
```css
repeating-linear-gradient(45deg, hsla(54,0%,88%,0.05) 0px, hsla(54,0%,88%,0.05) 45px,hsl(224,72%,31%) 45px, hsl(224,72%,31%) 50px)
```
This CSS Should show something like this:

image

Right now it shows only white color.

### PR DESCRIPTION
**Pull request by nicolasdeory**
Repeating linear gradients should now render properly (#59), and everything else should render like normal.

### CODE REVIEW

**Comment by bbenetskyy** [comment]
Just test on Android, it works as expected 👍

File: MagicGradients/Renderers/LinearGradientRenderer.cs [hunk 1]
```diff
@@ -19,12 +19,25 @@ public void Render(RenderContext context)
...
+
+            float computedOffset;
+            if (!_gradient.IsRepeating || orderedStops.LastOrDefault()?.Offset == null)
+            {
+                computedOffset = 1;
+                
+            } else
+            {
+                var angleRad = ToRad(_gradient.Angle);
+                var computedLength = Math.Sqrt(Math.Pow(info.Width * Math.Cos(angleRad), 2) + Math.Pow(info.Height * Math.Sin(angleRad), 2));
```
Comment (mgierlasinski) [inline MagicGradients/Renderers/LinearGradientRenderer.cs]: Could you please add comment what magic is happening here? Maybe link to math or something :)

Comment (nicolasdeory) [reply MagicGradients/Renderers/LinearGradientRenderer.cs:31]: I'm applying the Pythagorean Theorem to figure out the length of the gradient, which is needed to accurately calculate the `endPoint` for the gradient. Would you like me to add comments inside the code?

Comment (mgierlasinski) [reply MagicGradients/Renderers/LinearGradientRenderer.cs:31]: Yes please, simple comment in code with your first sentence + link will be fine :) If someone will look at the code later he can google details but will know what to search, thanks!

Comment (nicolasdeory) [reply MagicGradients/Renderers/LinearGradientRenderer.cs:34]: Done 👍

**Comment by bbenetskyy** [comment]
@nicolasdeory welcome to Magic Gradients Contributor 👏 🤝 👍

**Pull request merged**